Create a food

maxglassie opened the pull request:
The POST method works, including a set of validations. However, the validation message does not persist on the page.

maxglassie committed:
Creates GET all AJAX request

The AJAX request in food.js hits the localhost API endpoint and returns the data. It
sort of puts it in the table. The basic functionality is there.

maxglassie committed:
Completes AJAX request for all foods

Places the data into the table using jQuery on the foods.html page

maxglassie committed:
Builds HTML form for Creating New Foods

maxglassie committed:
workshop with Nate on AJAX requests

maxglassie committed:
Can create a food

maxglassie committed:
the POST method works

After a long time, it works on a basic level, but with no validations and
still needs to be encapsulated into a method. Now working on validations.

maxglassie committed:
Creates validation logic

It will flash the error message in the right place, below the name-field, but
it does not persist. Working on making it persist.

scmountain commented on lib/food.js:
```diff
@@ -1,10 +1,53 @@
+      url: API + '/api/v1/foods',
+      method: 'GET',
+    }).done(function(data){
+      for (var i = 0; i < data.length; i++) {
```
hand rolling .each nice work!

maxglassie committed:
Working on persisting validations

maxglassie committed:
Creates a food

The error message doesn't show up for very long, but it does update the food
and it does throw up the error.

scmountain commented on foods.html:
```diff
@@ -5,7 +5,44 @@
 
   <script src="main.bundle.js"></script>
+  <script ></script>
+  <script>
```
@maxglassie should we clean this out?
